scripts/exregional_run_prepstart.sh

```diff
@@ -626,6 +680,85 @@ EOF
    fi
 fi
 fi
+
+#
+#-----------------------------------------------------------------------
+#
+# Process FVCOM Data
+#
+#-----------------------------------------------------------------------
+#
+if [ "${USE_FVCOM}" = "TRUE" ]; then
+
+  set -x
+  latest_fvcom_file="${FVCOM_DIR}/${FVCOM_FILE}"
+  if [ ${HH} -gt 12 ]; then 
+    starttime_fvcom="$(date +%Y%m%d -d "${START_DATE}") 12"
+  else
+    starttime_fvcom="$(date +%Y%m%d -d "${START_DATE}") 00"
+  fi
+  for ii in $(seq 0 3)
+  do
+     jumphour=$((${ii} * 12))
+     fvcomtime=$(date +%Y%j%H -d "${starttime_fvcom}  ${jumphour} hours ago")
+     fvcom_data_fp="${latest_fvcom_file}_${fvcomtime}.nc"
+     if [ -f "${fvcom_data_fp}" ]; then
+       break 
+     fi
+  done
+#  latest_fvcom_file=$(ls -t ${FVCOM_DIR} | head -n1)
+#  fvcom_data_fp="${FVCOM_DIR}/${latest_fvcom_file}"
+  if [ ! -f "${fvcom_data_fp}" ]; then
+    print_err_msg_exit "\
+The file or path (fvcom_data_fp) does not exist:
+  fvcom_data_fp = \"${fvcom_data_fp}\"
+Please check the following user defined variables:
+  FVCOM_DIR = \"${FVCOM_DIR}\"
+  FVCOM_FILE= \"${FVCOM_FILE}\" "
+
+  else
+    cp_vrfy ${fvcom_data_fp} fvcom.nc
+
+#Format for fvcom_time: YYYY-MM-DDTHH:00:00.000000
+    fvcom_time="${YYYY}-${MM}-${DD}T${HH}:00:00.000000"
+#  fvcom_time="${DATE_FIRST_CYCL:0:4}-${DATE_FIRST_CYCL:4:2}-${DATE_FIRST_CYCL:6:2}T${CYCL_HRS[0]}:00:00.000000"
+#
```

Comment:
can we safely remove comments line 724 and 725?

Reply:
We can remove 724 and 725.